app/views/show.erb

```diff
@@ -0,0 +1,79 @@
+<div class="navbar navbar-static-top navbar-default">
+  <div class="container">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="/sources/<%= @client.identifier %>">Dashboard</a>
+    </div>
+    <div class="navbar-collapse collapse">
+      <ul class="nav navbar-nav navbar-right">
+        <li><a href="/sources/<%= @client.identifier%>/urls">Urls</a></li>
+        <li><a href="/sources/<%= @client.identifier%>/events">Events</a></li>
+      </ul>
+    </div>
+  </div>
+</div>
+
+<div class="container">
+
+  <h1 id="identifier"><%= @client.identifier%></h1>
+  <table class="table text-center" id="my_table">
+    <tr>
+      <td>
+        <h3>Average Response Time</h3>
+        <p><%= @client.payloads.average_response_time.round(3) %></p>
+      </td>
+      <td>
+        <h3>Max Response Time</h3>
+        <p><%= @client.payloads.max_response_time %></p>
+      </td>
+      <td>
+        <h3>Min Response Time</h3>
+        <p><%= @client.payloads.min_response_time %></p>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <h3>Most Frequent Request</h3>
+        <p><%= @client.request_types.most_frequent_request_type %></p>
+      </td>
+      <td>
+        <h3>All HTTP Verbs</h3>
+        <p> <%= @client.request_types.all_http_verbs.join(", ") %> </p>
```

Comment:
i know we talked about this before, but i'd suggest pulling this out to a method